modules/ossm-auto-route-enable.adoc

```diff
@@ -7,6 +7,14 @@
 
 By default, the `ServiceMeshControlPlane` resource automatically synchronizes the Gateway resources with OpenShift routes. Disabling the automatic route creation allows you more flexibility to control routes if you have a special case or prefer to control routes manually.
 
+== Skipping automatic route creation for specific cases
```

Comment:
"All titles and section headings must have an anchor ID. The anchor ID must be similar to the title or section heading."
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings

Reply:
Made the change, updated the PR, and generated a new preview build.